src/NRediSearch/AggregationResult.cs

```diff
@@ -9,6 +9,12 @@ namespace NRediSearch
     public sealed class AggregationResult
     {
         private readonly Dictionary<string, RedisValue>[] _results;
+
+        internal AggregationResult(RedisResult result, long cursorId) : this(result)
+        {
+            CursorId = cursorId;
+        }
+
         internal AggregationResult(RedisResult result)
```

Comment:
since this is `internal`, we can add an optional parameter without breaking the public API; suggest just one ctor: `internal AggregationResult(RedisResult result, long cursorId = -1) { ...}`